firstvoices/backend/models/media.py

```diff
@@ -270,6 +270,21 @@ def __str__(self):
         return f"{self.title} / {self.site} (Audio)"
 
 
+class Document(MediaBase):
+    class Meta:
+        verbose_name = _("Document")
+        verbose_name_plural = _("Document")
+        rules_permissions = {
+            "view": predicates.has_visible_site,
+            "add": predicates.is_at_least_assistant_or_super,
+            "change": predicates.is_at_least_assistant_or_super,
+            "delete": predicates.can_delete_media,
+        }
+
+    def __str__(self):
+        return f"{self.title} / {self.site} (Document)"
```

Comment:
[minor] For the sake of coverage, this could use a `test_representation` test in a new class `TestDocumentModel` within `test_media_models.py`. Take a look at the `test_representation` in `test_job_models` for a simple example